packages/editor/src/utils/editor.ts

```diff
@@ -126,7 +129,11 @@ export const createBlockHub: (
       if (editor.mode === 'page') {
         const defaultPageBlock = editor.querySelector('affine-default-page');
         assertExists(defaultPageBlock);
-        state.rect = defaultPageBlock.innerRect;
+        const rect = Rect.fromDOMRect(
+          defaultPageBlock.pageBlockContainer.getBoundingClientRect()
+        );
+        rect.height -= PAGE_BLOCK_PADDING_BOTTOM;
+        state.rect = rect;
```

Comment:
I think the height of `innerRect` is unexpected when the height of pageBlockContainer is less than the height of window, can you have a check? @fundon 


```ts
state.rect = defaultPageBlock.innerRect;
```

https://user-images.githubusercontent.com/18554747/237060103-0c84768b-6f36-4faa-8543-65dbb3d713c5.mov